flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
Is this needed now that you're installing a metainfo?

Reply:
The validator wants screenshots, but in this program there is really nothing to screenshot - it's non-interactive CLI.

Reply:
https://github.com/flathub/org.freedesktop.appstream-glib/issues/9